core/src/util/inputHandler.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "view/view.h"
+#include <memory>
+
+class InputHandler {
+
+public:
+    InputHandler(std::shared_ptr<View> _view);
+
+    void handleTapGesture(float _posX, float _posY);
+    void handleDoubleTapGesture(float _posX, float _posY);
+    void handlePanGesture(float _startX, float _startY, float _endX, float _endY);
+    void handlePinchGesture(float _posX, float _posY, float _scale, float _velocity);
+    void handleRotateGesture(float _posX, float _posY, float _radians);
+    void handleShoveGesture(float _distance);
+
+    void update(float _dt);
+
+private:
+
+    void setDeltas(float _zoom, glm::vec2 _translate);
+
+    void onEndGesture();
+
+    std::shared_ptr<View> m_view;
+
+    bool m_gestureOccured;
+
+    // fling deltas on zoom and translation
+    glm::vec2 m_deltaTranslate;
+    float m_deltaZoom;
+
+    /* Momentum config */
+
+    const float m_expDecrease = 0.95f;
```

Comment:
What would be the difference of using macros?  This number would be magic if used straight away in code without a name on it.